src/Conjure/Language/Expression/Op/Internal/Common.hs

```diff
@@ -236,6 +238,15 @@ operators =
     , ( BinaryOp L_TildeGeq     FNone   ,  400 )
     , ( UnaryPrefix L_Minus           , 2000 )
     , ( UnaryPrefix L_ExclamationMark , 2000 )
+    , ( UnaryPrefix L_Minus           , 2000 )
+    , ( UnaryPrefix L_ExclamationMark , 2000 )
+    ]
+--Functional operators that clash with other constructs so require no spaces
```

Comment:
Note: this means we are requiring that there will be no space between operator and the list of arguments.